src/components/functions/quotes.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const Quote = () => {
+  const [data, setData] = useState([]);
+  const [loaded, setLoaded] = useState(false);
```

Comment:
- [ ] According to project requirements there is an error state like this `const [hasError, setHasError] = useState(false)`  Please fix in order to make your project works as expected 👍

Reply:
Solved